docs/UserGuide.md

```diff
@@ -234,7 +246,7 @@ Sets the field as unique. Unique fields can not be used within [grammatical cons
 
 # Data Types
 
-DataHelix currently recognises four core data types: _string_, _datetime_, _integer_ and _decimal_.  It also recognises more complex data types which are extensions of these core types.  At present, all such data types are extensions of the _string_ type.
+DataHelix currently recognises three core data types: _string_, _datetime_, _integer_ and _decimal_. It also recognises more complex data types which are extensions of these core types.
```

Comment:
You say "three" data types but list 4.

Reply:
As discussed I will remove this line as it largly concerned with the internal type implementation and is potentially confusing to the reader.